.custom-gcl.reference.yml

```diff
@@ -0,0 +1,37 @@
+# The golangci-lint version used to build the custom binary.
+# Require.
+version: v1.56.2
+
+# the name of the custom binary.
+# Optional.
+# Default: custom-gcl
+name: custom-golangci-lint
+
+# The directory path used to store the custom binary.
+# Optional.
+# Default: .
+destination: ./my/path/
+
+# The list of the plugins to integrate inside the custom binary.
+plugins:
+  # a plugin from a Go proxy
+  - module: 'github.com/example/plugin3'
+    version: v1.2.3
```

Comment:
How about syntax `github.com/example/plugin3@v1.2.3` instead of splitting into two properties? This is what `go install` and `go run` [already use](https://go.dev/ref/mod#go-install).

Reply:
A local `path` is easily detectable when it starts with `/` or `./`. A remote path can be part of the specifier.

I don't mean to recommend `go install` but to use the common syntax among go tooling.

Reply:
`go.mod` does indeed not use the new syntax, but I guess it may do so eventually. Having a single string to specify a module is very beneficial to the whole ecosystem.

Reply:
We need the module name and the path.

I don't share your opinion about `go.mod` and the CLI syntax.

Reply:
Well, then complicate it however you like.

By the way, `version` is technically a incorrect term because it's a git tag that not necessarily has to hold a version. I hope it supports `latest`, which the go 1.16 syntax does.

Reply:
> Well, then complicate it however you like.

Please.

> By the way, version is technically a incorrect term because it's a git tag that not necessarily has to hold a version.

Technically speaking, it's not a git tag because `latest` is not a git tag.
Version is a generic term.